bcda/cli_test.go

```diff
@@ -150,6 +153,15 @@ func (s *CLITestSuite) TestImportCCLF9() {
 	args = []string{"bcda", "import-cclf9", "--file", "../shared_files/cclf/T.A0001.ACO.ZC9Y18.D181120.T1000010"}
```

Comment:
Can we have a test case for each file type?  Or does this test handle all scenarios?  
From the spec:
```The filename convention for the Medicare Shared Savings Program in Table 9 is P.A****.ACO.ZC9Y**.Dyymmdd.Thhmmsst.
The filename convention for the NGACO Model in Table 9 is P.V***.ACO.ZC9.Dyymmdd.Thhmmsst.
The filename convention for the CEC Model in Table 9 is P.CEC.ZC9.Dyymmdd.Thhmmsst.
The filename convention for the VT APM Model in Table 9 is P.F***.ACO.ZC9.Dyymmdd.Thhmmsst```

Reply:
This is a good flag.  However, I think the place for this test is in getCCLFFileMetaData below.  That has a regex that validates the file name.

Reply:
I thought BCDA was working with SSP ACOs only, at least at first, so we shouldn't need to test filename formats for the other models right now. @epfrankle , can you confirm?

Reply:
I also think more restrictive is better here.  If we did for some reason get a CCLFX file for a non SSP ACO we know we shouldn't have it and we should not import it.

Reply:
Confirmed that we are working with SSP ACOs to begin, and also want to leave ourselves available to work with other ACOs in the future.